frontend/ui/translator.lua

```diff
@@ -188,19 +188,6 @@ local Translator = {
         ssel = 0,  -- ?
         tsel = 0,  -- ?
         -- tk = "" -- auth token
-        dt = { -- what we want in result
-            "t",   -- translation of source text
-            "at",  -- alternate translations
-            -- Next options only give additional results when text is a single word
```

Comment:
That's probably why I ended up with the approach I had at first :sweat_smile: 

I can put the comments back here but I still think that `dt` should be added later on rather than here, so that it is clearer what's happening later on.

Reply:
We could put them all (with their documentation) where there are used, but that's a bit more than what this PR is about :)

Why there are here may not have been really thought (I may have just started translator.lua by copying wikipedia.lua), but the side effect of having it there is that people can user-patch it (like we suggested you to do :)) - so by hardcoded the replacement in the body of a function, the user patched stuff will be overridden.